doc/api/vm.md

```diff
@@ -98,7 +98,9 @@ changes:
     when `import()` is called. If this option is not specified, calls to
     `import()` will reject with [`ERR_VM_DYNAMIC_IMPORT_CALLBACK_MISSING`][].
     This option is part of the experimental modules API. We do not recommend
-    using it in a production environment.
+    using it in a production environment. If `--experimental-vm-modules` isn't
+    set, this callback will be ignored and Calls to `import()` will reject with
```

Comment:
I'm not sure "call to `import()`" is correct, as `import()` is not a function
```suggestion
    set, this callback will be ignored and `import()` expressions will reject with
```
alternatively
```suggestion
    set, this callback will be ignored and `import()` calls will reject with
```

Reply:
The `()` operator is named "Call" in spec terms, see the proposal for example: https://tc39.es/proposal-dynamic-import/#sec-import-calls - it's a `CallExpression`, so both are correct.

Reply:
I’m not a native speaker, so I might be wrong, but it seems to me there’s a nuance between `a call to import()` and `an import() call`.